minecraft_servers/thorma90/manifest.json

```diff
@@ -6,27 +6,27 @@
        "de"
     ],
     "social": {
-      "discord": "https://discord.gg/yRZEFzn5Kk",
-      "youtube": "https://www.youtube.com/@thorma9030",
-      "instagram": "thorma90"
+      "web": "https://thorma90.net",
```

Comment:
In maintenance; remove

Reply:
The website is no longer under maintenance!